src/frontend/packages/core/src/shared/components/list/list.component.html

```diff
@@ -201,4 +215,11 @@
       </ng-container>
     </div>
   </ng-container>
-</div>
\ No newline at end of file
+</div>
+
+<ng-template #refreshButton>
```

Comment:
Probably a merge artefact, can be deleted

Reply:
Good catch.